sdk/keyvault/azure-security-keyvault-jca/src/main/java/com/azure/security/keyvault/jca/implementation/KeyVaultClient.java

```diff
@@ -491,6 +491,9 @@ public byte[] getSignedWithPrivateKey(String digestName, String digestValue, Str
             } catch (IOException e) {
                 LOGGER.log(WARNING, "Failed to parse sign result response.", e);
             }
+        } else {
+            LOGGER.log(WARNING, "Can not get signature. It can be caused by missing 'sign' permission. To know how to add 'sign' permission. "
```

Comment:
`To know how to add 'sign' permission. See more from ...` -> `To know how to add 'sign' permission, see ...`